composer.json

```diff
@@ -4,12 +4,13 @@
   "type": "drupal-module",
   "license": "GPL-2.0-or-later",
   "require": {
-    "drush/drush": "^10.3",
+    "drush/drush": "^11",
     "drupal/purge": "^3.4",
+    "drupal/section_purge": "dev-3307199-drupal-10-compatibility",
     "drupal/section_purger": "dev-updates_to_compatible_with_drupal_9",
```

Comment:
This is getting removed, keeping this in composer until the `core.extention` gets cleaned up in 1.47.0, else it will throw error.

Reply:
This will be fine as long as code references use section_purge namespacing